selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -79,160 +80,163 @@
+    String PROJECT_EXPLORER_ITEM_TEMPLATE = "//div[@path='/%s']/div";
+    String COLOURED_ITEM_TEMPLATE =
+        "//div[@id='gwt-debug-projectTree']//div[@path='/%s']/descendant::div[@style='%s']";
+  }
 
   /**
-   * @param path path to item in Project Explorer
+   * Waits for visibility of the {@link WebElement} with specified {@code path} for a during {@code
+   * timeout} and gets this {@link WebElement}
+   *
+   * @param path path to item in format: 'RootFolder/src/main/java/org/qe/package'
    * @param timeout timeout in seconds
-   * @return found WebElement
+   * @return found {@link WebElement}
    */
-  private WebElement getProjectExplorerItem(String path, int timeout) {
-    return new WebDriverWait(seleniumWebDriver, timeout)
-        .until(visibilityOfElementLocated(By.xpath(format(PROJECT_EXPORER_ITEM_TEMPLATE, path))));
+  private WebElement waitAndGetItem(String path, int timeout) {
+    return seleniumWebDriverHelper.waitVisibility(
+        By.xpath(format(PROJECT_EXPLORER_ITEM_TEMPLATE, path)), timeout);
   }
 
   /**
-   * @param path path to item in Project Explorer
-   * @return item by path
+   * Waits for visibility of the {@link WebElement} with specified {@code path} and gets this {@link
+   * WebElement}
+   *
+   * @param path path to item in format: 'RootFolder/src/main/java/org/qe/package'
+   * @return found {@link WebElement}
    */
-  private WebElement getProjectExplorerItem(String path) {
-    return getProjectExplorerItem(path, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
+  private WebElement waitAndGetItem(String path) {
+    return waitAndGetItem(path, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
+  /** Waits for visibility and clicks on {@code Options} button in the project explorer area */
   public void clickOnProjectExplorerOptionsButton() {
-    redrawUiElementsWait
-        .until(
-            visibilityOfElementLocated(
-                By.xpath(
-                    "//div[@id='gwt-debug-navPanel']//div[@name='workBenchIconPartStackOptions']")))
-        .click();
+    seleniumWebDriverHelper.waitAndClick(
+        By.xpath("//div[@id='gwt-debug-navPanel']//div[@name='workBenchIconPartStackOptions']"),
+        REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
-  public void clickOnOptionsMenuItem(String menuID) {
-    redrawUiElementsWait
-        .until(visibilityOfElementLocated(By.xpath(format("//tr[@id='%s']", menuID))))
-        .click();
+  /**
+   * Waits for visibility and clicks on {@code item} with specified {@code id}
+   *
+   * @param item 'id' which stores in {@link ProjectExplorerOptionsMenuItem}
+   */
+  public void clickOnOptionsMenuItem(ProjectExplorerOptionsMenuItem item) {
+    seleniumWebDriverHelper.waitAndClick(
+        By.xpath(format("//tr[@id='%s']", item.get())), REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
   /** Clicks on project explorer tab and waits its appearance. */
-  public void openPanel() {
-    clickOnProjectExplorerTabInTheLeftPanel();
+  public void openAndWaitProjectExplorer() {
```

Comment:
`openAndWait` is enough because `openAndWaitProjectExplorer` duplicates class name **ProjectExplorer**.